heapoverflow/heapoverflow.json

```
@@ -0,0 +1,24 @@
+{
+    "key": "HeapOverflow",
+    "label": "HeapOverflow",
+    "display": "standalone",
+    "access": "external",
+    "image": {
+        "name": "heapoverflow",
```

Image will be built + pushed to `cheesehub/heap-overflow`:
https://github.com/cheese-hub/Heap-Overflow/pull/1/files#diff-3f5366f6d6df3ec1179e5efadc6f350bfa88eebf4e2da589b4d94ccb85ae5e94R40

Image name here should match:
```suggestion
        "name": "cheesehub/heap-overflow",
```